src/shared/components/ContextMenu/ContextMenu.style.ts

```diff
@@ -19,28 +19,7 @@ export const StyledContainer = styled.div<ContainerProps>`
   color: ${colors.white};
   padding: ${sizes(2)};
   word-break: break-all;
-`
```

Comment:
Cannot add a comment where I'd like to but there is unneeded import at the top of the file